plugins/foreman_ansible/3.x/index.md

```diff
@@ -0,0 +1,307 @@
+<p>
+Our callback is installed with Ansible itself, change <code>/etc/ansible/ansible.cfg</code> to contain
+</p>
+
+{% highlight bash %}
+[defaults]
+callback_whitelist = foreman
+{% endhighlight %}
+
+<p>
+and the callback will be enabled after that.
+</p>
+
+<p>
+Of course, the callback cannot know automatically where is Foreman located and which certificates to use in its requests.
+</p>
+</div>
+
+<div class="ansible_version ansible_version_25" style="display:none;">
+<p>
+To configure it, add a section <code>[callback_foreman]</code> at the end of <code>/etc/ansible/ansible.cfg</code> like this:
+</p>
+
+{% highlight bash %}
+[callback_foreman]
+url = 'https://foreman.example.com'
+ssl_cert = /etc/foreman-proxy/ssl_cert.pem
+ssl_key = /etc/foreman-proxy/ssl_key.pem
+verify_certs = /etc/foreman-proxy/ssl_ca.pem
+{% endhighlight %}
+</div>
+
+<div class="ansible_version ansible_version_21" style="display:none;">
+<p>
+We don't really test these versions, however if you need to use an old version of Ansible, you could do the following:
+</p>
+
+<ul>
+  <li>
+  Change your <code>/etc/ansible/ansible.cfg</code> to contain the following options:
+  </li>
+
+{% highlight bash %}
+callback_plugins = ~/.ansible/plugins/callback_plugins/
+bin_ansible_callbacks = True
+{% endhighlight %}
+
+  <li>
+  And run this line to download the callback file in the appropriate location:
+  </li>
+
+{% highlight bash %}
+wget https://raw.githubusercontent.com/ansible/ansible/stable-2.2/lib/ansible/plugins/callback/foreman.py -O ~/.ansible/plugins/callback_plugins/foreman.py
+{% endhighlight %}
+</ul>
+</div>
+
+<div class="ansible_version ansible_version_22 ansible_version_21" style="display:none;">
+
+<p>
+You need to configure it via the following environment variables (the variables need to be available to the user running the callback. Usually <code>foreman-proxy</code>):
+</p>
+
+<ul>
+  <li>
+    FOREMAN_URL: the URL of your Foreman installation (default "http://localhost:3000")
+  </li>
+  <li>
+    FOREMAN_SSL_CERT: The public key when using SSL client certificates (default "/etc/foreman/client_cert.pem")
+  </li>
+  <li>
+    FOREMAN_SSL_KEY: The private key when using SSL client certificates (default  "/etc/foreman/client_key.pem")
+  </li>
+  <li>
+    FOREMAN_SSL_VERIFY: Use *False* to disable certificate checks. You can also set it
+    to use the local CA bundle "True", or a path to a CA file for a custom CA.
+  </li>
+</ul>
+
+<p>
+Since the 'foreman' and 'foreman-proxy' are 'no-login' users, you should set these in the <code>EnviromentFile</code> used by systemd, <code>/etc/sysconfig/foreman</code> and <code>/etc/sysconfig/foreman-proxy</code>. An alternative for other users is to put them in their <code>.bashrc</code> as <code>export FOREMAN_URL=yoururl</code>. In most distributions environment variables set in <code>/etc/environment</code> will be shared among all users.
```

Comment:
We no longer support running directly from Foreman so `<code>/etc/sysconfig/foreman</code>` is no longer needed